RecoLocalTracker/SiPixelRecHits/src/PixelCPETemplateReco.cc

```diff
@@ -286,14 +293,15 @@ PixelCPETemplateReco::localPosition(DetParam const & theDetParam, ClusterParam &
       //		templQbin_ );
 
 
-      float dchisq;
-      float templProbQ_;
       std::vector< SiPixelTemplateStore2D > thePixelTemp2D_;
       SiPixelTemplate2D::pushfile(ID, thePixelTemp2D_);
       SiPixelTemplate2D templ2D_(thePixelTemp2D_);
       	
-      theClusterParam.ierr =
-	SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,
+      theClusterParam.ierr = -123;
+      /*
+      float dchisq;
+      float templProbQ_;
+      SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,
```

Comment:
I am fine with commenting out.
Deleting  is perhaps too severe, somebody has been working on this splitting code for many months.
So deleting it would mean starting from scratch. I know one recover it from gut but in practice
code which is delete gets forgotten very fast,
D.

On 23 Feb 2016, at 21:45, Slava Krutelyov <notifications@github.com<mailto:notifications@github.com>> wrote:


In RecoLocalTracker/SiPixelRecHits/src/PixelCPETemplateReco.cc<https://github.com/cms-sw/cmssw/pull/13382#discussion_r53846830>:

>        std::vector< SiPixelTemplateStore2D > thePixelTemp2D_;
>        SiPixelTemplate2D::pushfile(ID, thePixelTemp2D_);
>        SiPixelTemplate2D templ2D_(thePixelTemp2D_);
>
> -      theClusterParam.ierr =
> -     SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,
> +      theClusterParam.ierr = -123;
> +      /*
> +      float dchisq;
> +      float templProbQ_;
> +      SiPixelTemplateSplit::PixelTempSplit( ID, theClusterParam.cotalpha, theClusterParam.cotbeta,


@dkotlins<https://github.com/dkotlins>
this PR has other changes. The ones in MagneticField are the most likely ones that triggered SIM step change.

Are you OK with effectively turning off this splitting step here in the way it's done (comment it out or, equivalently, remove from the compilation)?
If I understood Vincenzo correctly, this part of the code if commented out will not compile anymore.

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/13382/files#r53846830>.